### ISSUE TITLE
Séparer la partie cozytouch de la partie domoticz

### ISSUE THREAD

**Issue by LudovicRousseau** [OBone-git/Cozytouch-domoticz#48]
J'ai un chauffe eau Malicio 3 de Thermor.
Je n'utilise pas domoticz.
Mais je veux quand même récupérer les données de mon chauffe eau pour les injecter dans ma base influxdb.

Pour l'instant `cozytouch.py` est un code monolithique qui fait tout. Je propose de séparer la partie qui interroge cozytouch de la partie qui interagie avec domoticz.
C'est peut-être ce qui est déjà prévu dans l'idée de plugin dont il est question en https://github.com/OBone-git/Cozytouch-domoticz/issues/40#issuecomment-1032296120

J'ai sauvagement hacker le code pour : 
- ignorer la partie domoticz
- migrer en Python 3 (grace à l'outil 2to3)

La bonne nouvelle est que je peux m'authentifier au serveur cozytouch:
```
 POST-> https://ha110-1.overkiz.com/enduser-mobile-web/enduserAPI/login | userId=****&userPassword=**** : 200
Authentification serveur cozytouch OK
Connexion serveur Cozytouch reussie
```

Est-ce qu'une séparation cozytouch/domoticz est prévue ?
Si non est-ce qu'elle serait utile au projet ?

Au départ j'étais parti sur https://github.com/Cyr-ius/cozytouchpy mais l'authentification échoue.
Je ne sais pas dans quel projet investir mes efforts : cozytouchpy ou Cozytouch-domoticz ?

**Comment by OBone-git** [comment]
Bonjour,

En #40 il est question de porter le script existant en une version moins sauvage et plus évolutive sous Python3, en une version que Domoticz supportera en tant que plugin natif, en non pas en fonction tournant indépendamment en arrière.

Peut être que les données d'authentification de [https://github.com/Cyr-ius/cozytouchpy](url) ne sont pas à jour (url & clef) Elles ont évolué récemment et j'ai fait un récent pull à ce sujet.
Une fois les bonnes informations à disposition il est très simple d'y accéder. 

[https://github.com/Cyr-ius/cozytouchpy](url) me parait plus "propre" bien que plus compliqué. Il est certain que j'irai y piocher des infos, bien que cette base ne doit pas être évidente à adapter avec une passerelle entre les deux systèmes Cozytouch & Domoticz.

**Comment by Turbox35** [comment]
Bonjour,
Utilisateur de domoticz également, je cherche à récupérer mes informations cozytouch sans justement dépendre de domoticz.
Idéalement en docker avec envoi sur serveur MQTT avec l'auto discovery, afin d'avoir une installation facile et de ne pas dépendre des mis à jour de domoticz.

**Comment by LudovicRousseau** [comment]
@Turbox35 tu peux regarder mon code sur https://github.com/LudovicRousseau/Cozytouch-domoticz/tree/wip (branche wip). J'ai enlevé tout le code domoticz qui ne me sert pas.

L'utilisation est simple:
```
$ ./cozytouch.py 
ChauffeEau core:BottomTankWaterTemperatureState=51.3,core:RemainingHotWaterState=155,modbuslink:MiddleWaterTemperatureState=54.9,modbuslink:PowerHeatElectricalState=0
```
Il faut un fichier `ma_config.py` qui contient :
```
login="ton login"
password="ton password"
```

**Comment by soyuka** [comment]
@LudovicRousseau t'as réussi a faire qqch sur le Malicio 3 au final ?

**Comment by LudovicRousseau** [comment]
@soyuka oui, mon code est en production depuis plus d'un an et ça marche
https://github.com/LudovicRousseau/Cozytouch-domoticz/tree/wip
